src/portal/lib/src/service/interface.ts

```diff
@@ -290,25 +290,35 @@ export enum VulnerabilitySeverity {
 
 export interface VulnerabilityBase {
   id: string;
-  severity: VulnerabilitySeverity;
+  severity: string;
   package: string;
   version: string;
 }
 
 export interface VulnerabilityItem extends VulnerabilityBase {
-  link: string;
-  fixedVersion: string;
+  links: string[];
+  fix_version: string;
   layer?: string;
   description: string;
 }
 
 export interface VulnerabilitySummary {
-  image_digest?: string;
-  scan_status: string;
-  job_id?: number;
-  severity: VulnerabilitySeverity;
-  components: VulnerabilityComponents;
-  update_time: Date; // Use as complete timestamp
+    report_id?: string;
+    mime_type?: string;
+    scan_status?: string;
+    severity?: string;
+    duration?: number;
+    summary?: SeveritySummary;
```

Comment:
It is a little bit tricky to use it as summary.summary. It is because you want to keep consistent with backend api naming rule?

Reply:
I use "VulnerabilitySeverityMetrics" ,"SeveritySummary" and "VulnerabilitySummary" types to distinguish them. But for backend api , they are all called"summary".